src/domains/wallet/pages/WalletDetails/hooks/use-wallet-options.ts

```diff
@@ -96,7 +96,7 @@ const getRegistrationOptions = (wallet: Contracts.IReadWriteWallet, t: TFunction
 
 	if (wallet.balance() > 0 && !wallet.isLedger() && isRestoredAndSynced(wallet)) {
 		if (
-			!isMultiSignature(wallet) &&
+			(isMainsailNetwork(wallet.network()) || !isMultiSignature(wallet)) &&
```

Comment:
I'll add a card to handle this through psdk, as it would be helpful to have this based purely on the network manifest as that also indicates which musig transactions are supported